docs/usage/configuration-options.md

```diff
@@ -2558,8 +2558,7 @@ You must use the `!/ /` syntax, like this:
 
 ### changelogUrl
 
-Use this field to set the changelog URL for a package, including overriding any existing one.
-Using this field we can specify the exact URL to fetch changelogs from.
+This doesn't help with _fetching_ the changelogs, but if you configure it then Renovate will include a link to this URL in the PR body, so users can click through to read the changelog.
```

Comment:
```suggestion
Sometimes Renovate does not show the correct changelog for a package.
As a workaround for this problem, you can give Renovate the URL to the changelog with the `changelogUrl` config option.
When set, Renovate will put a _link_ to the changelogs in the Renovate PR body.

Renovate does _not_ show the complete changelogs from the `changelogUrl` in its PR body text, you only get the URL from Renovate.
To read the changelogs you must use the link.
```

My goals with this text are to explain that:

- `changelogUrl` is a workaround for missing changelogs
- you only get a _link_ to the changelog, not the full changelogs

We already have a note admonition below after the codeblock to explain that setting `changelogUrl` does not help Renovate _fetch_ the changelogs. So I'm dropping that bit of info in my rewrite.